geometry_msgs/Point[] victim_locations
---
geometry_msgs/Point[] victim_visiting_order
